runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptionsTest.java

```diff
@@ -181,6 +182,7 @@ public void testDefaultInvalidGcpTempLocation() {
   }
 
   @Test
+  @Ignore
```

Comment:
This should have something descriptive, ideally a link to a JIRA (which in turn references the test), or if it's no longer useful can be removed.